web/templates/web/children-list.html

```diff
@@ -13,6 +12,43 @@
 {% endblock %}
 {% block content %}
     <script>
+        $(document).ready(function() {
+            $('.datepicker').datepicker({})
+            .on('change', function () {
+               var age = getAge(this);
+               if (document.getElementById('age_calc')) {
+                   document.getElementById('age_calc').innerHTML = `Age: ${age}`;
+               } else {
+                   var x = document.createElement("p");
+                   x.setAttribute("id", "age_calc");
+                   x.setAttribute("class", "age_format")
+                   var t = document.createTextNode(`Age: ${age}`);
+                   x.appendChild(t);
+                   document.getElementsByClassName("form-group")[1].insertBefore(x,document.getElementById("id_birthday"));
+               }
+           });
+        });
+        function getAge(dateValue) {
```

Comment:
How this is working, if younger than a month, I am showing age in days, if younger than a year, i show age in months, rounded down, if younger than two years, I show age as 1 year x months, and older than two years, show age in years.  Not exact but pretty close.